python/pipeline/stack.py

```diff
@@ -985,7 +986,8 @@ class Surface(dj.Computed):
     -> PreprocessedStack
     -> shared.SurfaceMethod
     ---
-    surface_im        : longblob          # Matrix of calculated depth for each pixel in stack
+    guessed_points    : blob              # Array of guessed depths stored in (z,y,x) format
```

Comment:
is blob big enough?

Reply:
I guess I can error on the side of caution and make it a longblob.